meteor/lib/api/studios.ts

```diff
@@ -32,6 +33,6 @@ export interface UIStudio {
 
 	settings: IStudioSettings
 
-	routeSets: Record<string, StudioRouteSet>
-	routeSetExclusivityGroups: Record<string, StudioRouteSetExclusivityGroup>
+	routeSets: ObjectWithOverrides<Record<string, StudioRouteSet>>
```

Comment:
Another thought, that is not related to your changes but could be improved as part of it.

Does the ui which uses this really need the full `StudioRouteSet`, or does it just need a few properties? I am mostly wondering if it needs the `routes` property, which can be a large structure.
It would be nice to reduce this in this publication if possible, but that is not at all related to what you are changing, so can be left for someone else to do later.

Reply:
Yes, that makes sense. But might be something to look at later, if the general structure of settings gets an overhaul.